tox.ini

```diff
@@ -1,5 +1,14 @@
 [tox]
-envlist = lint, fmt
+envlist = py, lint, fmt
+
+[testenv]
+description = run unit tests
+deps =
+    pytest>=7
+    torch
```

Comment:
why are torch and transformers here ? these deps are already installed as part of project installation. test environment should install anything else missing like pytest

Reply:
torch and transformers should come from here https://github.com/foundation-model-stack/fms-hf-tuning/blob/main/requirements.txt

Reply:
It seems the tox always creates an isolated virtual env to run the test. I could remove them but ran into error such as
`E   ModuleNotFoundError: No module named 'torch'`

Reply:
for projects that do not have setup.py the recommendation is to install from requirements https://web.archive.org/web/20150112223937/https://blog.schwuk.com/2014/03/19/using-tox-django-projects/ 

but pls chekc with @hickeyma on best practice around this, we do not want to manage same list of dependencies multiple times by copy pasting them into files

Reply:
sorry I missed your comment above, ya the snippet you posted with `-r requirements.txt` looks better @tedhtchang ! Thanks, we just dont want to copy out deps from requirements

Reply:
@Ssukriti I think the requirements.txt will also be gone after pyproject.toml get merged.

Reply:
We can ask @jbusche to keep the requirements.txt for now so we can get this merged. After he merged his PR, we can think about how to remove the requirements.txt. I am thinking something like:
```
skip_install = true
whitelist_externals = poetry
commands =
    poetry install
    poetry run pytest
```